ui-v2/app/templates/settings.hbs

```diff
@@ -26,13 +11,13 @@
             </p>
             <form>
                 <fieldset>
-                    <label class="type-text">
-                        <span>ACL Token</span>
-                        {{ input type='password' value=item.token name="token" }}
-                        <em>The token is sent with requests as the <code>X-Consul-Token</code> HTTP header parameter. This is used to control the ACL for the web UI.</em>
-                    </label>
+                    <div class="type-toggle">
+                      <label>
+                          <input type="checkbox" name="client[blocking]" checked={{if item.client.blocking 'checked' }} onchange={{action 'change'}} />
+                          <span>Enable Catalog realtime updates (blocking queries)</span>
```

Comment:
I tried to be specific what this does as I've no idea when this will land in a release. We may be able to add in blocking queries to the rest of the app before then, we may not.